config/initializers/0_clowder_rails.rb

```diff
@@ -0,0 +1,17 @@
+require 'clowder-common-ruby/rails_config'
+
+if ClowderCommonRuby::Config.clowder_enabled?
```

Comment:
Let's make sure that `Settings` is available for the current context:
```suggestion
if ClowderCommonRuby::Config.clowder_enabled? && defined?(Settings)
```
cc @vkrizan so you sleep better :wink: